src/components/NFTcard/nftcard.tsx

```diff
@@ -11,12 +14,16 @@ export interface NftCardProps {
   placeBid: (clickedCard: NFT) => void
 }
 
+
 const NTFCard: React.FC<NftCardProps> = ({ cardContent, navigateToCard, addToWhiteList, placeBid }) => {
+  const { t } = useTranslation()
+  
+
   return (
     <div>
       <Card>
         <Media>
-          <CardImg src={cardContent.image} />
+          <CardImg src={`${process.env.REACT_APP_IPFS_URL}${cardContent.image}`} />
```

Comment:
kindly use this service instead https://github.com/StartFi/startfi-interface/blob/7cf25ac7c8e2969381ebe71e9a24140902204784/src/utils/uriToHttp.ts

Reply:
Done